spec/examples/hello/hello_client.rb

```diff
@@ -1,23 +1,26 @@
 require_relative './hello_services_pb'
 
 class HelloClient
-  def initialize
-    @client = Hello::Hello::Stub.new('localhost:8000', :this_channel_is_insecure)
+  def initialize(host = 'localhost:8000')
+    @client = Hello::Hello::Stub.new(host, :this_channel_is_insecure)
   end
 
-  def send_message(msg, client_stream: false, server_stream: false, metadata: {})
+  def send_message(msg, client_stream: false, server_stream: false, metadata: {}, timeout: 0.1)
```

Comment:
Spec requests to localhost fail immediately, but when the tests try to hit a fake outside service we need a timeout to keep things fast.